### ISSUE TITLE
Running finetuned network for images from internet

### ISSUE THREAD

**Issue by Pattinson** [valentin-deschaintre/Guided_fine_tuning_SVBRDF#1]
Hi,

Thank you very much for putting all code, training data, and trained weight on the web. It was super useful. I am trying to understand your work and extend it. When going through the code, I got some confusion. The first is about the test script, the second is about image size and the third is about generating exampler. 

1). I have been trying to go through your code for guided fine-tuning paper. I am not able to run the network on test mode for the images from internet images. I think it is because of some input parameter mismatch. Do you have any sample commands to run the network on test mode?

The process I follow is: using the provided weight, I ran the fine-tuning script. It creates a fine-tuned network for a given exemplar. Then I tried to run the network using the weight of the fine-tuned network for image download from the internet. 

I am able to retrain the network and run fine-tuning using the command provided on the README file. 

 2). Secondly, I am confused about the image size. Your model is trained to work for [256x256](https://github.com/valentin-deschaintre/Guided_fine_tuning_SVBRDF/blob/master/model.py#L34) images right? The training data size is 2048x(4x2048), fine-tune network script crop image with 512x512, with exampler being 256x256. Also, I saw that image is resized to 588 which I believe to get 2 splits from this code (` widthSplit = int(np.ceil(width / strideSize)) - 1`). My assumption till now is that image is cropped to make 256x256 and then merge when generating output from fine-tune network.

 3) For the user acquired data, What would be the program flow from beginning to end. The flow I have been assuming is like this. First, we need to feed flask images to your previous paper's model (here, first phase model) to get 4 maps, then merge those to form a single exampler file. After that run `testscript_finaltraining.py` script to get fine-tune network. Also, if there is a single script to do this all then that would be helpful.

**Comment by valentin-deschaintre** [comment]
Hi,
Here are the answer to the questions as I understand them.

1) The test mode is for a given usecase where you have all the ground truth. I think what you are looking for is eval mode? I don't think I save the weights of the fine tuned network by default though, so you will have to do this save yourself.

2) The model is trained for 512 x 512 images. The examplar resize is simply to fit in the resolution the original network was trained with. I resize it to 588 so that at each training step the examplar is slightly different (as it will crop in these 588x588 a 512x512 image) 

3) This flow sounds good, I however do not have such an end to end script. It should not be too difficult to plug the previous network in testScript_finalTraining.py if you need to though.

Best,

**Comment by joeyism** [comment]
@Pattinson I'm trying to do what you're doing for step 3. Did you happen to make an end to end script for that?
